Steps/SetupPythonPreReqs.yml

```diff
@@ -23,7 +23,13 @@ steps:
       versionSpec: ">=3.10.6"
       architecture: x64
 
-- script: sudo pip install ${{ parameters.pip_requirement_files }} --upgrade
+- script: |
+    python3 -m venv .venv
+    source .venv/bin/activate
+  displayName: Setup Python Virtual Environment (Linux)
+  condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
+
+- script: pip install ${{ parameters.pip_requirement_files }} --upgrade
   displayName: Install and Upgrade pip Modules (Linux)
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))

```

Comment:
We might as well drop the separate Linux step then right?

Reply:
I prefer them separate so we can better isolate which operation failed (1) creating the virtual environment or (2) install the pip modules in the virtual environment.

Reply:
I meant the "Install and Upgrade pip Modules" since it is now identical to the windows step below

Reply:
Sorry, misunderstood. Yeah, I'll merge those.

Reply:
Done in this PR commit https://github.com/microsoft/mu_devops/pull/22/commits/42f505f868835945bda5e96379061187cdb9bedc